lib/determinator/retrieve/dynaconf.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+
+module Determinator
+  module Retrieve
+    # A class which loads features from Dynaconf server
+    class Dynaconf
+      # @param :host [String] The host for local Dynaconf server
+      # @param :client [String] Faraday client instance, defaults to a new instance
+      def initialize(host:, client: default_client)
+        @host = host
+        @client = client
+      end
+
+      def retrieve(feature_id)
+        url = "http://#{@host}/scopes/florence-#{feature_id}/feature"
+
+        payload = @client.get(url).body
```

Comment:
Non blocking question: I assume that all the info we need is in the body, that is we don't really care about the HTTP status code.

Reply:
I've now explicitly make it to check for 200 otherwise silently fail. Or any explicitly raised error is captured with extra notice.